templates/provider/kcm/templates/rbac/user-facing/clusteripam-editor.yaml

```diff
@@ -0,0 +1,15 @@
+# permissions for end users to edit managementbackups.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    k0rdent.mirantis.com/aggregate-to-global-admin: "true"
```

Comment:
Why should the permissions be set to global? Shouldn't the permissions have per namespace granularity?